test/ParentConstructor.test.ts

```diff
@@ -0,0 +1,66 @@
+import { ISerializableAsync, SerializableAsync, serialize, type, validate } from "@js-soft/ts-serval"
+import { expect } from "chai"
+
+class NewSerializable {
+    public static from<T extends NewSerializable>(): T {
+        const ctor = (this as any).prototype.constructor
+        const obj = new ctor()
+
+        return obj
+    }
+}
+
+class TestSerializableClass extends NewSerializable {
+    private readonly property = "test"
+
+    public getProperty(): string {
+        return this.property
+    }
+}
+
+interface IParentItem extends ISerializableAsync {}
+
+@type("ParentItem")
+class ParentItem extends SerializableAsync implements IParentItem {
+    public static async from(value: IParentItem): Promise<ParentItem> {
+        return await super.fromT(value)
+    }
+}
+
+interface IChildItem extends IParentItem {
+    test: string
+}
+
+@type("ChildItem")
+class ChildItem extends ParentItem implements IChildItem {
+    @serialize()
+    @validate()
+    public test: string
+}
+
+export class ParentConstructorTest {
+    public static init(): void {
+        describe("querying the parent constructor in a static function", function () {
```

Comment:
Irgendwie hast du dich mit den Test Descriptions verzettelt. ;)

> querying the parent constructor in a static function ... should create a ChildItem instead of a ParentItem when the ParentItem overrides the from method

Das Querien des Parent Konstrukturs erstellt ja keine Items

Zumal sich die Testbeschreibungen ja sowieso eher auf die Verwendersicht konzentrieren sollten und nicht darauf, wie es implementiert ist ("querying the parent constructor in a static function" ist eine Beschreibung der Implementierung)